vowpalwabbit/multilabel.cc

```diff
@@ -135,11 +137,10 @@ void print_update(vw& all, bool is_test, example& ec)
     if (is_test)
       label_string << " unknown";
     else
-      for (size_t i = 0; i < ec.l.multilabels.label_v.size(); i++) label_string << " " << ec.l.multilabels.label_v[i];
+      for (unsigned int i : ec.l.multilabels.label_v) label_string << " " << i;
 
     std::stringstream pred_string;
-    for (size_t i = 0; i < ec.pred.multilabels.label_v.size(); i++)
-      pred_string << " " << ec.pred.multilabels.label_v[i];
+    for (unsigned int i : ec.pred.multilabels.label_v) pred_string << " " << i;
```

Comment:
Use the same type as is contained
```suggestion
    for (uint32_t i : ec.l.multilabels.label_v) { label_string << " " << i; }
```

Reply:
changed to uint32_t.  change to reference label look like a mistake.